layouts/shortcodes/alert.html

```diff
@@ -0,0 +1 @@
+<div class="alert {{ with .Get `type` }}alert-{{ . }}{{ else }}alert-warning{{ end }}">{{ .Inner }}</div>
```

Comment:
adding dashes to inner might be a good idea to reduce whitespace `{{- .Inner -}}` or we could also wrap it with something like ```{{- with .Inner -}}...yourexistingcode...{{- end -}}``` so it reduces whitespace and if you did `{{< alert >}}{{< alert />}}` nothing would be output.

Reply:
I'd also recommend parsing the content as Markdown so that the residual raw HTML is no longer necessary